mbedtls-sys/build/build.rs

```diff
@@ -57,21 +62,16 @@ impl BuildConfig {
                 if have_feature("custom_printf") {
                     try!(writeln!(f, "int mbedtls_printf(const char *format, ...);"));
                 }
-                if have_feature("custom_has_support") {
-                    try!(writeln!(
-                        f,
-                        "int mbedtls_aesni_has_support( unsigned int what ) __attribute__((weak));"
-                    ));
-                    try!(writeln!(
-                        f,
-                        "int mbedtls_padlock_has_support( int feature ) __attribute__((weak));"
-                    ));
-                }
                 if have_feature("custom_threading") {
                     try!(writeln!(f, "typedef void* mbedtls_threading_mutex_t;"));
                 }
+		if have_feature("custom_has_support") {
```

Comment:
Use the standard feature=>define mechanism

Reply:
I am sorry, I have not fully understood this.
Would there be a better way to express below?
```
if have_feature("custom_has_support") {
    try!(writeln!(f, "#define CUSTOM_HAS_AESNI"));
    try!(writeln!(f, "#define CUSTOM_HAS_PADLOCK"));
}
```

Reply:
Yes, don't manually write anything. Use the mechanisms from config.rs